payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayPaymentMethodLauncher.kt

```diff
@@ -259,7 +260,24 @@ class GooglePayPaymentMethodLauncher internal constructor(
         fun onResult(result: Result)
     }
 
+    /**
+     * Error codes representing the possible error types for [Result.Failed].
+     * See the corresponding [Result.Failed.error] message for more details.
+     */
+    @Target(AnnotationTarget.PROPERTY, AnnotationTarget.VALUE_PARAMETER, AnnotationTarget.TYPE)
+    @IntDef(INTERNAL_ERROR, DEVELOPER_ERROR, NETWORK_ERROR)
+    annotation class ErrorCode
```

Comment:
These are meant to be very high level to define the error message to be shown to the user.